proxy/src/inbound.rs

```diff
@@ -33,7 +34,9 @@ impl<B> Inbound<B> {
 
 impl<B> Recognize for Inbound<B>
 where
-    B: tower_h2::Body + 'static,
+    B: tower_h2::Body + Send + 'static,
+    B::Data: Send,
+    <B::Data as ::bytes::IntoBuf>::Buf: Send,
```

Comment:
This is fine for now, but it re-enforces my thought that `IntoBuf` should not be used as part of the `Data` trait.

Reply:
I agree!